File: VB/WindowsApplication3/MVVM/ViewModels/OrdersViewModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports MVVMSample.MVVM.Data

Namespace MVVMSample
    Public Class OrdersViewModel
        Inherits BaseViewModel(Of Order)

        Public Sub New()
            Entities = DataAccessHelper.GetOrders()
        End Sub
    End Class
End Namespace

File: CS/WindowsApplication3/MVVM/ViewModels/OrdersViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using MVVMSample.MVVM.Data;

namespace MVVMSample {
    public class OrdersViewModel : BaseViewModel<Order> {
        public OrdersViewModel() {
            Entities = DataAccessHelper.GetOrders();
        }
    }
}
